src/components/logout/Logout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import vector from '../../shared/images/vector.svg';
+
+function Logout() {
+  const navigate = useNavigate();
+
+  const callback = () => {
+    navigate('/sign_in');
+  };
+
+  const logoutUser = () => {
+    localStorage.removeItem('token');
```

Comment:
Maybe should we defined constant 'TOKEN' instead of having such string?  If yes we could change this in another places too. What do you think?

Reply:
Great idea!